pkg/minishift/addon/parser/addon_parser.go

```diff
@@ -180,12 +180,25 @@ func (parser *AddOnParser) parseCommands(scanner *bufio.Scanner) ([]command.Comm
 }
 
 func createMetaData(header []string) (addon.AddOnMeta, error) {
-	regex, _ := regexp.Compile(`^# ?(.*):(.*)`)
-	metaMap := make(map[string]string)
+	regex, _ := regexp.Compile(`^# ?([a-zA-Z-]*):(.*)`)
+	metaMap := make(map[string]interface{})
+	var key string
+	var value []string
 	for _, line := range header {
 		matches := regex.FindAllStringSubmatch(line, -1)
+		if len(matches) == 0 && key == "Description" {
```

Comment:
As mentioned earlier, I 'd rather also see the regexps as a contant, as this makes them named and more understandable in their purpose. If this is extracted as a constant, consider the regex patterns too, please.